inginious/frontend/templates/course.html

```diff
@@ -68,6 +68,9 @@ <h3> {{_("Last tried exercises") }}</h3>
                     <i class="fa fa-user-times fa-fw"></i>&nbsp; {{ _("Unregister from this course") }}
                 </a>
             {% endif %}
+            <a class="list-group-item list-group-item-action list-group-item-info" href="{{ get_homepath() }}/user_settings/{{ course.get_id() }}">
+                <i class="fa fa-cogs fa-fw"></i>&nbsp; {{ _("Course Settings") }}
```

Comment:
Currently as the administrator is still not a course student (we should change that in the future), the link is not shown to the admin. But it should, because if not accessible it won't allow the admin to design and tests the task relying on that feature. I'd rather put this link before the unregister one to avoid misclicks.